docs/admin/high-availability.md

```diff
@@ -0,0 +1,63 @@
+# High Availability
+
+High Availability (HA) mode solves for horizontal scalability and automatic failover
+within a single region. When in HA mode, Coder continues using a single Postgres
+endpoint. [GCP](https://cloud.google.com/sql/docs/postgres/high-availability), [AWS](https://docs.aws.amazon.com/prescriptive-guidance/latest/saas-multitenant-managed-postgresql/availability.html),
+and other cloud vendors offer fully-managed HA Postgres services that pair
+nicely with Coder.
+
+For Coder to operate correctly, every node must be within 10ms of each other
+and Postgres. We make a best-effort attempt to warn the user when inter-coder
+latency is too high, but if requests start dropping, this is one metric to investigate.
```

Comment:
Since we surface the database latency, should we document the endpoint/dashboard to check this?

Reply:
We should do that once the Admin Settings page is in. cc @kylecarbs